tools/import/importservice.js

```diff
@@ -62,7 +62,7 @@ export default class ImportService {
     const $this = this;
 
     const poll = async () => {
-      if ($this.busy) {
+      if ($this.busy || !$this.apiKey) {
```

Comment:
I think this check should be moved further down.

Reply:
```
if (this.config.poll && this.apiKey)
```

Reply:
Has to be before line 70.  Or do you mean inside "queryJob"?

Reply:
`poll` is an inner function. Add the check on Line 88 before `await poll();` is called for the first time.